experiments/kompanion/cmd/export/export.go

```diff
@@ -116,20 +221,22 @@ func run(_ *cobra.Command, _ []string) {
 		log.Fatalf("Failed to get preferred resources: %s", err)
 	}
 
-	resources := []*metav1.APIResource{}
 	for _, apiResourceList := range apiResourceLists {
 		if !strings.Contains(apiResourceList.GroupVersion, "cnrm.") {
```

Comment:
Nit: `.cnrm.cloud.google.com/` might be less likely to accidentally collect the wrong thing